cmake/FindrocBLAS.cmake

```diff
@@ -0,0 +1,67 @@
+#==========================================================================
+#  Copyright (C) Codeplay Software Limited
```

Comment:
For the new files you are creating, could you please add this to the Copyright notice as well 
(above #  Copyright (C) Codeplay Software Limited)?
#  Copyright 2020-2022 Intel Corporation.

So, it will look like this:

#==========================================================================
#  Copyright 2020-2022 Intel Corporation
#  Copyright (C) Codeplay Software Limited
rest of the copyright...

Reply:
Sure, we'll incorporate the new line into the copyright